drake/examples/Acrobot/test/paramEstSyntheticData.m

```diff
@@ -2,28 +2,31 @@
 
 tmp = addpathTemporary(fullfile(pwd,'..'));
 
-% Synthetic parameter estimation mode
-% 'base':       Base case parameter estimation - intial params = true params
-% 'paramerr':   w/ parameter error but no measurement noise
-% 'measnoise':  w/ parameter error and w/ measurement noise
-% 'delay':      w/ parameter error and w/ measurement noise and delay (Not complete)
-mode = 'measnoise';
+%% CONFIGURATION
+% Introduce parameter error into estimation
+hasParamErr = true;
+% Standard deviation of the parameter percent error
+paramstd = .20;
+
+% Introduce measurement noise into estimation
+hasMeasNoise = true;
+% Standard deviation of error for theta1,theta2,theta1dot,theta2dot,theta1doubledot,theta2doubledot
+% noisestd = sqrt([.000001, .000001, .0000001, .0000001, 0, 0,]);
+noisestd = sqrt([1e-3, 1e-3, 1e-11, 1e-11, 0, 0,]);
+
+% Introduce delay into estimation (Not complete)
+delay = false;
 
 % Parameter Estimation model
 % 'dynamic'     = use dynamic model - requires qdd
 % 'energetic'   = use energetic model - doesn't require qdd
```

Comment:
Yah, it is more like units of the cost function - although technically torque could also be force if we have a linear actuator as opposed to a rotary one.  Maybe change it to "force_torque" just to be general enough.